README.md

```diff
@@ -63,7 +72,7 @@ The return value from the promise utils functions works very well with TypeScrip
 
 The full `promise-utils` API consists of the following functions:
 
-- `go(asyncFn)` - Executes the `asyncFn` and returns a response of type `GoResult`
+- `go(asyncFn, options)` - Executes the `asyncFn` and returns a response of type `GoResult`
 - `goSync(fn)` - Executes the `fn` and returns a response of type `GoResult`
```

Comment:
Mhh, I wonder if goSync should not accept the `PromiseOptions` as well - but I do not see a good enough use for that so let's NOT do that.